internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -97,7 +97,6 @@ func (r *AgentServiceConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		reason := fmt.Sprintf("Invalid name (%s)", instance.Name)
 		msg := fmt.Sprintf("Only one AgentServiceConfig supported per cluster and must be named '%s'", agentServiceConfigName)
 		r.Log.Info(fmt.Sprintf("%s: %s", reason, msg), req.NamespacedName)
-		r.Recorder.Event(instance, "Warning", reason, msg)
```

Comment:
Yeah. I meant to drop it from the test, not entirely :sunglasses: . We do want to emit an event when we are completely ignoring a CR (like here).

Reply:
Ok. I misunderstood what you meant. I will add the recorder back.